test/services/FeedProcessor.test.ts

```diff
@@ -371,11 +371,15 @@ describe('FeedProcessor', () => {
           calculatorRepository.find.withArgs('Identity').returns(new IdentityCalculator());
 
           const feeds1: Feeds = {
-            TEST: feedFactory.build({inputs: [feedInputFactory.build({fetcher: {name: 'WrongFetcher'}})]}),
+            TEST: feedFactory.build({
+              inputs: [feedInputFactory.build({fetcher: {name: 'WrongFetcher' as FetcherName}})],
+            }),
```

Comment:
I assume that this test doesn't make sense anymore, before we were relying on strings and but with your improvements we accept only enums and there is no room for this kind of error. I would remove this kind of unit tests.

Reply:
Yeah, I think we are safe form this error now. Even from feeds file we ensure with the feeds-schema that we are using only the enums

Reply:
Done